hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/Store.java

```diff
@@ -46,6 +46,8 @@
   int PRIORITY_USER = 1;
   int NO_PRIORITY = Integer.MIN_VALUE;
 
+  int getBlockSize();
```

Comment:
Is this necessary? Its available on the ColumnFamilyDescriptor. Do we need to add new method on Store?

Was this change always here or did it just show up in recent amendments to PRs (I don't remember seeing it before but probably just me).

Thanks.

Reply:
this was new change in the latest commit of this PR, mainly it's used for `HMobStore` when calling `createWriterInTmp`. but let me change it using  `getStoreContext().getBlockSize()` to avoid adding a new interface.

Reply:
Yes. Please. Then I'll +1 this nice PR. Thanks.